sdk/tests/storage/blobs/test_az_storage_blobs.c

```diff
@@ -32,6 +35,23 @@ void test_storage_blobs_init(void** state)
       NULL)));
 }
 
+void test_storage_blobs_init_nonnull_options(void** state);
+void test_storage_blobs_init_nonnull_options(void** state)
```

Comment:
We do we need to define the function like this first before implementing it?

Doesn't look like we do this elsewhere, unless I am mistaken. And the tests are just static.

Reply:
We do. Could be in other places, but we do. This is to avoid warning on one of the compilers that we define a function without prior declaration.